app/scripts/modules/core/help/helpContents.js

```diff
@@ -157,8 +157,6 @@ module.exports = angular.module('spinnaker.core.help.contents', [])
     'pipeline.config.quickPatchAsg.rollingPatch': '<p>Patch one instance at a time vs. all at once.</p>',
     'pipeline.config.quickPatchAsg.skipUpToDate': '<p>Skip instances which already have the requested version.</p>',
     'pipeline.config.jenkins.propertyFile': '<p>(Optional) Configures the name to the Jenkins artifact file used to pass in properties to later stages in the Spinnaker pipeline.</p>',
-    'pipeline.config.bake.package': '<p>The name of the package you want installed (without any version identifiers).</p>' +
-      '<p>If your build produces a deb file named "myapp_1.27-h343", you would want to enter "myapp" here.</p>',
```

Comment:
Thanks @anotherchrisberry for clarification. I made the change (locally) as you mentioned. But, the string in netflixHelpContents.registry.js is getting displayed on my setup. Somehow it thinks it is NetFlix version of spinnaker. Is that the default behavior?